src/azul/logging.py

```diff
@@ -30,15 +78,20 @@ def configure_test_logging(*loggers):
 
 
 def _configure_non_app_logging(*loggers):
-    logging.basicConfig(format='%(asctime)s %(levelname)-7s %(threadName)s: %(message)s')
+    handler = logging.StreamHandler()
+    handler.addFilter(LambdaLogFilter())
+    logging.basicConfig(format=log_format,
```

Comment:
With this the non-app logs like the unit test logs all carry that fake request ID and omit the thread name.

We don't want that.

This has taken so long that I forgot why we even need the LambdaLogFilter. Please raise in PL.

Reply:
From PL:
`make local` should resemble what occurs on AWS Lambda.
Keep the original log format for unit tests.